src/octoprint/plugins/backup/__init__.py

```diff
@@ -454,6 +393,22 @@ def bodysize_hook(self, current_max_body_sizes, *args, **kwargs):
         # max upload size of 1GB for the restore endpoint
         return [("POST", r"/restore", 1024 * 1024 * 1024)]
 
+    # Exported plugin helpers
+    def create_backup_helper(self, exclude=None, backup_file=None):
+        """
+        Trigger a backup from a plugin or other internal call
+        Args:
+             exclude (list): Identifiers of data folders to exclude
+        """
+        if exclude is None or not (type(exclude) == list):
```

Comment:
Better use `isinstance(exclude, list)` here, or we rule out anything that inherits from `list`.